alerting/ingest-alerting/variables.tf

```diff
@@ -0,0 +1,32 @@
+# Global
+
+variable "suffix" {
+  description = "Webhook name suffix, will precede the notif period"
```

Comment:
> will precede the notif period

it is false, currently it defines the full integration name.

Reply:
Copy paste from https://github.com/claranet/terraform-signalfx-integrations/blob/master/alerting/proxy-alerting/variables.tf#L4, fixed.